src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -203,11 +200,11 @@ private void trim(CorfuPayloadMsg<TrimRequest> msg, ChannelHandlerContext ctx, I
     LoadingCache<LogAddress, LogData> dataCache;
     long maxCacheSize;
 
-    private AbstractLocalLog localLog;
+    private AbstractLocalLog localLog = null;
 
     public static long maxLogFileSize = Integer.MAX_VALUE;  // 2GB by default
 
-    private final ConcurrentHashMap<UUID, AbstractLocalLog> streamLogs = new ConcurrentHashMap<>();
```

Comment:
Why would field ever need to be not final? Even if you wanted to reset the function, you could just call `streamLogs.clear()`.